opslevel/resource_opslevel_domain_test.go

```diff
@@ -0,0 +1,61 @@
+package opslevel
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccExampleResource(t *testing.T) {
```

Comment:
shouldn't the name of this call out that its for the domain resource?

Reply:
💯